SuperBuild/python_configure_python_launcher.cmake

```diff
@@ -126,6 +126,21 @@ set(PYTHONLAUNCHER_PYTHONPATH_INSTALLED
   "<APPLAUNCHER_DIR>/../${Slicer_BIN_DIR}/Python"
   )
 
+# Add paths necessary to import VTK
+if (APPLE)
+  list(APPEND PYTHONLAUNCHER_PYTHONPATH_INSTALLED
+    "${PYTHONHOME}/lib/Slicer-4.9"
+    )
+elseif(MSVC)
```

Comment:
We very rarely use MSVC variable in Slicer CMake files. Instead, we mostly use UNIX, APPLE, and WIN32. So, it could be if (APPLE) ... elseif (UNIX) ... else() ... endif()

Reply:
50/50, I chose wrong :-). Will fix.